uk.co.compucorp.civicrm.hrcore/CRM/HRCore/CMSData/UserAccountInterface.php

```diff
@@ -45,4 +45,13 @@ public function enable($contactData);
    * @return boolean
    */
   public function isUserDisabled($contactData);
+
+  /**
+   * Finds user by email
+   *
+   * @param $mail
+   *
+   * @return mixed
+   */
+  public function findUserByEmail($mail);
```

Comment:
Do we need this here? The `CRM_HRCore_CMSData_Role_DrupalRoleService` class does not extend this class.